src/features/unified-share-modal/ContactRestrictionNotice.js

```diff
@@ -103,6 +103,7 @@ const ContactRestrictionNotice = ({
                     {...restrictionNoticeMessage}
                     values={{ count: restrictedExternalContactCount, email: firstEmail }}
                 />
+                &nbsp;
```

Comment:
Is it possible to replace `&nbsp;` with styles change ?

Reply:
Yes, but I think it's preferable to use the `&nbsp;` in order to get the correct space width based on the current font size, etc.